src/lib/ngsiNotify/QueueNotifier.cpp

```diff
@@ -60,7 +60,16 @@ int QueueNotifier::start()
 *
 * QueueNotifier::sendNotifyContextRequest -
 */
-void QueueNotifier::sendNotifyContextRequest(NotifyContextRequest* ncr, const std::string& url, const std::string& tenant, const std::string& xauthToken, const std::string& fiwareCorrelator, NotificationFormat notifyFormat)
+void QueueNotifier::sendNotifyContextRequest
+(
+  NotifyContextRequest*            ncr,
+  const std::string&               url,
+  const std::string&               tenant,
+  const std::string&               xauthToken,
+  const std::string&               fiwareCorrelator,
+  RenderFormat                     renderFormat,
+  const std::vector<std::string>&  attrsFilter
```

Comment:
attrsOrder? (in this case it seem clear, as this method is like the one in the parent Notifeir class, which uses attrsOrder)